agent/utils/add_metalog_option.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
```

Comment:
Could we consider making an `agent/lib/pbench` module path, and placing all these "utils" there as modules, and then have actual commands in `agent/util-scripts` invoke them?

Reply:
That wouldn't be a problem, though `agent/lib` seems to be just perl scripts, didn't want to mix those.